documentation/faq.asciidoc

```diff
@@ -252,6 +252,65 @@ Please use this only as the last resort solution.
 
 Sometimes while doing experiments (or when a connector was misconfigured at the start) it is necessary to remove the connector offsets to start with a clean state.
 
+There are two ways to reset the offsets depending on your Kafka Connect version:
+
+* xref:#reset-offsets-kafka-3.6[]
+* xref:#reset-offsets-kafka-3.5[]
+
+[id=reset-offsets-kafka-3.6]
+=== With Kafka Connect 3.6 or later
+
+Kafka Connect introduced a set of dedicated REST API endpoints for managing connector offsets as part of KIP-875.
```

Comment:
maybe just for convenience add a link to KIP-875 (https://cwiki.apache.org/confluence/display/KAFKA/KIP-875%3A+First-class+offsets+support+in+Kafka+Connect)?